[#286]add ansible playbook to run sysctl setup

jhd2best opened the pull request:
create an ansible playbook to run sysctl setup on any node.

LeoHChen commented:
ansible has a sysctl module, please use the module

https://docs.ansible.com/ansible/latest/modules/sysctl_module.html

jhd2best commented:
**Test Result:**
PASS

**Test Log:**
ansible-playbook -i inventory/haodi.hosts playbooks/sysctl-setup.yml -e 'inventory=test user=hmy'

PLAY [test] ******************************************************************************************************************************

TASK [Gathering Facts] *******************************************************************************************************************
ok: [52.191.166.139]

TASK [kernel tuning] *********************************************************************************************************************
changed: [52.191.166.139] => (item={'name': 'fs.file-max', 'value': '2097152'})
changed: [52.191.166.139] => (item={'name': 'vm.swappiness', 'value': '10'})
changed: [52.191.166.139] => (item={'name': 'vm.dirty_ratio', 'value': '60'})
changed: [52.191.166.139] => (item={'name': 'vm.dirty_background_ratio', 'value': '2'})
changed: [52.191.166.139] => (item={'name': 'kernel.sched_migration_cost_ns', 'value': '5000000'})

TASK [general network security options] **************************************************************************************************
changed: [52.191.166.139] => (item={'name': 'net.ipv4.tcp_synack_retries', 'value': '2'})
changed: [52.191.166.139] => (item={'name': 'net.ipv4.ip_local_port_range', 'value': '2000 65535'})
changed: [52.191.166.139] => (item={'name': 'net.ipv4.tcp_rfc1337', 'value': '1'})
changed: [52.191.166.139] => (item={'name': 'net.ipv4.tcp_syncookies', 'value': '1'})
changed: [52.191.166.139] => (item={'name': 'net.ipv4.tcp_fin_timeout', 'value': '15'})
changed: [52.191.166.139] => (item={'name': 'net.ipv4.tcp_keepalive_time', 'value': '300'})
changed: [52.191.166.139] => (item={'name': 'net.ipv4.tcp_keepalive_probes', 'value': '5'})
changed: [52.191.166.139] => (item={'name': 'net.ipv4.tcp_keepalive_intvl', 'value': '15'})

TASK [tuning network performance] ********************************************************************************************************
changed: [52.191.166.139] => (item={'name': 'net.core.rmem_default', 'value': '31457280'})
changed: [52.191.166.139] => (item={'name': 'net.core.rmem_max', 'value': '33554432'})
changed: [52.191.166.139] => (item={'name': 'net.core.wmem_default', 'value': '31457280'})
changed: [52.191.166.139] => (item={'name': 'net.core.wmem_max', 'value': '33554432'})
changed: [52.191.166.139] => (item={'name': 'net.core.somaxconn', 'value': '8096'})
changed: [52.191.166.139] => (item={'name': 'net.core.netdev_max_backlog', 'value': '65536'})
changed: [52.191.166.139] => (item={'name': 'net.core.optmem_max', 'value': '25165824'})
changed: [52.191.166.139] => (item={'name': 'net.ipv4.tcp_max_syn_backlog', 'value': '8192'})
changed: [52.191.166.139] => (item={'name': 'net.ipv4.tcp_mem', 'value': '786432 1048576 26777216'})
changed: [52.191.166.139] => (item={'name': 'net.ipv4.udp_mem', 'value': '65536 131072 262144'})
changed: [52.191.166.139] => (item={'name': 'net.ipv4.tcp_rmem', 'value': '8192 87380 33554432'})
changed: [52.191.166.139] => (item={'name': 'net.ipv4.udp_rmem_min', 'value': '16384'})
changed: [52.191.166.139] => (item={'name': 'net.ipv4.tcp_wmem', 'value': '8192 65536 33554432'})
changed: [52.191.166.139] => (item={'name': 'net.ipv4.udp_wmem_min', 'value': '16384'})
changed: [52.191.166.139] => (item={'name': 'net.ipv4.tcp_max_tw_buckets', 'value': '1440000'})
changed: [52.191.166.139] => (item={'name': 'net.ipv4.tcp_tw_reuse', 'value': '1'})
changed: [52.191.166.139] => (item={'name': 'net.ipv4.tcp_fastopen', 'value': '3'})
changed: [52.191.166.139] => (item={'name': 'net.ipv4.tcp_window_scaling', 'value': '1'})

PLAY RECAP *******************************************************************************************************************************
52.191.166.139             : ok=4    changed=3    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0

jhd2best commented:
> ansible has a sysctl module, please use the module
> 
> https://docs.ansible.com/ansible/latest/modules/sysctl_module.html

updated
